examples/simulated_edgesystem_graphite.py

```diff
@@ -55,16 +55,17 @@ def simulated_sampling_function():
 
 if __name__ == '__main__':
 
-    system = SimulatedSystem(config['SystemName'])
+    edgesystem = SimulatedEdgeSystem(config['EdgeSystemName'])
```

Comment:
Variable name can be changed to edge_system everywhere better readability of code.